content_collections/_updates/sel4-tutorials/camkes-3.8.x.md

```diff
@@ -16,18 +16,18 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 - `interrupts`: Remove sample output that doesn't appear in practice.
 - `hello-camkes-2`: Update exit text test to match actual output.
 - `mcs`: reduce spinner budget for final task to ensure timeout behavior happens correctly.
-- `dynamic-4`: Correctly initalize a stack variable.
+- `libraries-4`: Correctly initalize a stack variable.
```

Comment:
In these older release notes we shouldn't change the name -- in the version it refers to, they were still called `dynamic-`.